### ISSUE TITLE
Use of 'drop_path' and 'geo_mask'

### ISSUE THREAD

**Issue by KL4805** [HKUDS/OpenCity#11]
It seems that 'drop_path' and 'geo_mask' are always disabled in OpenCity, it this the case?

**Comment by LZH-YS1998** [comment]
Yes, if you want to use drop_path, you can set a specific value for it.
